integration-test/src/test/java/org/cloudfoundry/package-info.java

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I copied this from other modules to enable creating `org.immutable.Value.Immutable` classes in the tests using the same defaults. The immutable objects I added to the tests are for invoking the Log Cache test application's endpoint to emit certain Log Cache events.

Reply:
I see that there's one in the service broker module, so having one makes sense. But it seems it should be within the logcache module rather than the parent level? Again, happy to be educated!